packages/react-components/src/components/styles/Captions.style.ts

```diff
@@ -21,6 +21,33 @@ export const displayNameClassName = mergeStyles({
   lineHeight: _pxToRem(16)
 });
 
+/* @conditional-compile-remove(rtt) */
+/**
+ * @private
+ */
+export const isTypingClassName = (theme: ITheme): string => {
+  return mergeStyles({
+    backgroundColor: theme.palette.themeLighter,
+    color: theme.palette.themeDarker,
+    borderRadius: _pxToRem(4),
```

Comment:
nit: can we calculate the size here and use that? this function is best used when there are weird sizing like the 11px below.